frontend/catalyst/passes/pass_api.py

```diff
@@ -354,6 +354,7 @@ def _API_name_to_pass_name():
         "cancel_inverses": "remove-chained-self-inverse",
         "merge_rotations": "merge-rotations",
         "ions_decomposition": "ions-decomposition",
-        "to_ppr": "to_ppr",
-        "commute_ppr": "commute_ppr",
+        "to_ppr": "convert-clifford-t-to-ppr",
+        "commute_ppr": "commute-clifford-t-ppr",
```

Comment:
revert this I assume
```suggestion
        "to_ppr": "to_ppr",
        "commute_ppr": "commute_ppr",
```